doc/releases/mimic.rst

```diff
@@ -6,8 +6,162 @@ this is not the final release yet, consider testing this on your testing and non
 production clusters for a preview of upcoming features and please report all the
 bugs to the tracker.
 
+Major Changes from Luminous
+---------------------------
+
+- *Dashbaord*:
```

Comment:
@liewegas too late, as this PR has already been merged, but there's typo here: "Dashbaord".

I also added a blurb about the new Dashboard to the PendingReleaseNotes document, maybe that provides some additional color.